### ISSUE TITLE
Can't build because of bitvec broken patching

### ISSUE THREAD

**Issue by notdanilo** [RustCrypto/elliptic-curves#307]
Are you aware of this issue?
https://github.com/bitvecto-rs/bitvec/issues/105

**Comment by tarcieri** [comment]
Yes, it's patched in `elliptic-curve` v0.9.3 (see https://github.com/RustCrypto/traits/pull/550) however that's only usable with prereleases of the crates in this repo.

Perhaps I should backport the fix to `elliptic-curve` v0.8 as well.

**Comment by notdanilo** [comment]
That would be great if you could fix it in v0.8 too. Otherwise, I have to track down all the dependency chain and report it to each one of them to get this fixed in our project.
